openlibrary/accounts/model.py

```diff
@@ -597,19 +595,17 @@ def s3auth(cls, access_key, secret_key):
         from openlibrary.core import lending
         url = lending.config_ia_s3_auth_url
         try:
-            req = urllib.request.Request(url, headers={
+            response = requests.get(url, headers={
                 'Content-Type': 'application/json',
                 'authorization': 'LOW %s:%s' % (access_key, secret_key)
             })
-            f = urllib.request.urlopen(req)
-            response = f.read()
-            f.close()
-        except urllib.error.HTTPError as e:
-            try:
-                response = e.read()
-            except simplejson.decoder.JSONDecodeError:
-                return {'error': e.read(), 'code': e.code}
-        return simplejson.loads(response)
+            if not response.ok:
+                response.raise_for_status()
```

Comment:
```suggestion
            response.raise_for_status()
```
`response.raise_for_status()` will be a no-op if there is no error.